charmhelpers/core/unitdata.py

```diff
@@ -173,13 +173,25 @@ class Storage(object):
     """
     def __init__(self, path=None, keep_revisions=False):
         self.db_path = path
+        # The following is a hack to work around a bug in which both the
+        # ops framework libraries and charmhelpers end up using the same
+        # DB path, which leads to conflicts.
+        # See: https://bugs.launchpad.net/charm-ceph-mon/+bug/2005137
+        db_suffix = '.unit-state.db'
```

Comment:
It's not a 'suffix'; it's a hidden file.  e.g. `db_suffix` isn't quite right; `db_file` would be more accurate.